au4/src/projectscene/iplaycursorcontroller.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "modularity/imoduleinterface.h"
+#include "view/timeline/timelinecontext.h"
+
+namespace au::projectscene {
+class IPlayCursorController : MODULE_EXPORT_INTERFACE
+{
+    INTERFACE_ID(processing::IPlayCursorController)
+
+public:
+    virtual ~IPlayCursorController() = default;
+
+    virtual void setTimelineContext(TimelineContext* context) = 0;
+    virtual void moveCursor(int pixels) = 0;
+    virtual void seekToX(double x) = 0;
+    virtual void updatePositionX(double x) = 0;
```

Comment:
I believe we need to do things a little differently: 
* IPlayCursorController - this is the main controller for the play cursor and it should only operate on time (there should be no x coordinates, pixels, etc.)
* PlayCursorViewController - This is an adapter for view, and its main task is to convert coordinates, pixels into time and back, and pass this time to the main playcursor controller.